fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/derby/DerbyTranslator.java

```diff
@@ -163,4 +165,30 @@ public String getUrl(Properties connectionProperties) {
     public boolean clobSupportsInline() {
         return false;
     }
+    
+    /* (non-Javadoc)
+     * @see com.ibm.fhir.database.utils.api.IDatabaseTranslator#getType()
+     */
+    @Override
+    public DbType getType() {
+        return DbType.DERBY;
+    }
+
+    /* (non-Javadoc)
+     * @see com.ibm.fhir.database.utils.api.IDatabaseTranslator#dualTableName()
+     */
+    @Override
+    public String dualTableName() {
+        return "SYSIBM.SYSDUMMY1";
+    }
+
+    /* (non-Javadoc)
+     * @see com.ibm.fhir.database.utils.api.IDatabaseTranslator#selectSequenceNextValue(java.lang.String, java.lang.String)
+     */
+    @Override
+    public String selectSequenceNextValue(String schemaName, String sequenceName) {
+        String qname = DataDefinitionUtil.getQualifiedName(schemaName, sequenceName);
+        return "SELECT NEXT VALUE FOR " + qname + " FROM SYSIBM.SYSDUMMY1";
```

Comment:
In other. spots you use a final String, seems a bit inconsistent.

Reply:
In general we should do a better job of using final where appropriate throughout the code. When usage is obvious, it's less of an issue. But we could certainly benefit from it in more complex methods.

Reply:
made it final in the next commit